internal/config/nsmgr/nsmgr.go

```diff
@@ -69,6 +69,9 @@ func (mgr *NamespaceManager) NewPodNamespaces(cfg *PodNamespacesConfig) ([]Names
 		if !ok {
 			return nil, errors.Errorf("Invalid namespace type: %s", ns.Type)
 		}
+		if ns.Host {
+			arg += "=host"
```

Comment:
may want to share a constant between pinns and here but could be done later.

Reply:
yeah especially if pinns is taken out of this repo. It's not super trivial to do so, so I'll leave it out for now.